src/main/java/com/moodybluez/enterprise/MoodyBluezController.java

```diff
@@ -1,17 +1,52 @@
 package com.moodybluez.enterprise;
 
+import com.moodybluez.enterprise.dao.MoodDAO;
+import com.moodybluez.enterprise.dto.Mood;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
 
 @Controller
-public class MoodyBluezController {
+public class MoodyBluezController implements ErrorController {
 
     /**
      * Handles the root endpoint and returns index.html
-     * @return
+     * @return Returns a thymeleaf template
      */
     @RequestMapping("/")
     public String index() {
         return "index";
     }
+
+    /**
+     * Returns the json representation of a Mood by id
+     * @param id integer id of a mood
+     * @return Mood in JSON format
+     * @author Stephen Meckstroth
```

Comment:
param and return are good, but the author tag is somewhat deprecated.

That was handy back in the days before good version control (believe it or not, there were days like that.  Now, though, we can see change history and the person who did it in version control, so no need for the author tag.